src/server/scripts/EasternKingdoms/Karazhan/karazhan.h

```diff
@@ -23,25 +23,22 @@
 
 enum DataTypes
 {
-    TYPE_ATTUMEN                    = 1,
-    TYPE_MOROES                     = 2,
-    TYPE_MAIDEN                     = 3,
-    TYPE_OPTIONAL_BOSS              = 4,
-    TYPE_OPERA                      = 5,
-    TYPE_CURATOR                    = 6,
-    TYPE_ARAN                       = 7,
-    TYPE_TERESTIAN                  = 8,
-    TYPE_NETHERSPITE                = 9,
-    TYPE_CHESS                      = 10,
-    TYPE_MALCHEZZAR                 = 11,
-    TYPE_NIGHTBANE                  = 12,
+    DATA_ATTUMEN                    = 1,
```

Comment:
That would have been the right thing to do, if it had been done by the original scripter. Are you sure that the values are only counters and can be set to start from 0 in the scripts they are used in? (I don't know, it just feels edgy.)